src/docs/src/install/unix.rst

```diff
@@ -152,6 +152,8 @@ You should have the following installed:
 * `help2man                     <http://www.gnu.org/s/help2man/>`_
 * `Python (>=3.6) for docs and tests      <http://python.org/>`_
 * `Python Sphinx (>=1.1.3)      <http://pypi.python.org/pypi/Sphinx>`_
+* Java (required for `nouveau`)
```

Comment:
Is there a minimum / recommended version for them?

Also wonder what release packaging is going to look for various OS combinations we support (RHEL-clones, Debian, Windows).